wire-ios-request-strategy/Sources/Payloads/Processing/PayloadProcessing+MLSMessageSendingStatusTests.swift

```diff
@@ -0,0 +1,59 @@
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+@testable import WireRequestStrategy
+
+class PayloadProcessing_MLSMessageSendingStatusTests: MessagingTestBase {
+
+    let domain = "example.com"
+
+    override func setUp() {
+        super.setUp()
+
+        syncMOC.performGroupedBlockAndWait {
+            self.otherUser.domain = self.domain
+        }
+    }
+
+    func testThatItAddsFailedToSendRecipients() throws {
+        try self.syncMOC.performGroupedAndWait { _ in
+            // given
+            guard let message = try self.groupConversation.appendText(content: "Test message") as? ZMClientMessage else {
+                XCTFail("Failed to add message")
+                return
+            }
+
+            XCTAssertEqual(message.failedToSendRecipients?.count, 0)
+
+            // When
+            var failedToSendUsers: [QualifiedID] = []
+            if let qualifiedID = self.otherUser.qualifiedID {
+                failedToSendUsers = [qualifiedID]
+            }
```

Comment:
Perhaps it would be good to make sure that we populate this array.

Reply:
Could you more clarify this comment?
I use failedToSendUsers array to create MLSMessageSendingStatus payload.